src/Features/Core/Portable/SymbolSearch/Windows/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -75,21 +74,20 @@ public ValueTask UpdateContinuouslyAsync(string source, string localSettingsDire
 
             // We were the first ones to try to update this source.  Spawn off a task to do
             // the updating.
-            return new Updater(this, logService, source, localSettingsDirectory).UpdateInBackgroundAsync(cancellationToken);
+            return new Updater(this, source, localSettingsDirectory).UpdateInBackgroundAsync(cancellationToken);
         }
 
         private sealed class Updater
         {
             private readonly SymbolSearchUpdateEngine _service;
             private readonly string _source;
             private readonly DirectoryInfo _cacheDirectoryInfo;
-            private readonly ISymbolSearchLogService _logService;
+            private readonly ISymbolSearchLogService _logService = SymbolSearchLogService.Instance;
```

Comment:
💭 Might as well get rid of the interface at this point?

Reply:
Yeah, agreed.  Will do.